inference-engine/src/inference_engine/include/ie/ie_plugin_config.hpp

```diff
@@ -283,6 +283,16 @@ DECLARE_CONFIG_VALUE(CPU_THROUGHPUT_AUTO);
  */
 DECLARE_CONFIG_KEY(PERF_COUNT);
 
+/**
+ * @brief The name for setting work mode in MULTI device plugin option.
+ *
+ * It is passed to Core::SetConfig(), this option should be used with value only:
+ * PluginConfigParams::AUTO or PluginConfigParams::MULTI
+ */
+DECLARE_CONFIG_KEY(WORK_MODE);
```

Comment:
as these are assumingly not supposed to be used directly by a user, shall we move them to the ie_internal_plugin_config.hpp and use the CONFIG_KEY_INTERNAL macro

Reply:
good idea, will update